### PR TITLE
Initialized a basic demo scheme for TNO

### PR DESCRIPTION
**Pull request by Sam-TNO**

### CODE REVIEW

**Comment by Sam-TNO** [comment]
Hi, I'm trying to create a simple demo scheme to be used by TNO. We want to make a slightly more complex demo, but started with this very simple credential. Did I properly create this pull request? Any help would be appreciated. Thanks!

File: TNO/Issues/first_test/description.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	<IssuerID>tnodemo</IssuerID>
+	<CredentialID>demofortno</CredentialID>
+	<Description>
+		<en>A short demo to test functionality, to be expanded later</en>
+		<nl>Een kleine demo om de functionaliteit te testen, om later uitgebreid te worden</nl>
+	</Description>
+	<IssueURL>
+		<en>[TBD]</en>
+		<nl>[TBD]</nl>
+	</IssueURL>
```
Comment (ivard) [inline TNO/Issues/first_test/description.xml:20]: If you don't want to set a `IssueURL`, so can also completely omit it. Then the app will not trip over a non-parsable URL.

**Review by ivard — COMMENTED** [review]
It looks fine to me. You only need to sign the scheme as a whole by running `irma scheme sign` and commit the new `index`, `index.sig` and `timestamp` files.

**Comment by Sam-TNO** [comment]
Thank you for your quick reply! I think that issue is resolved now

**Review by ivard — APPROVED** [review]

**Review by bobhageman — APPROVED** [review]

**Pull request merged**